main.py

```diff
@@ -0,0 +1,42 @@
+"""
+Created on Fri Apr 30 10:42:58 2021
+
+@author: olaf_
+"""
+
+from pandas import read_excel
+from modules.cellPostprocessing import postprocess
+from modules.method import Method
+from modules.myplots import Myplots
+
+#%% Load & postprocess cell data
```

Comment:
It is good practice to put the code in a `if __name__ == "__main__":` [block](https://docs.python.org/3/library/__main__.html). Otherwise you might inadvertently run the code when someone imports your code as a package.